frog/imports/client/GraphEditor/Activities.js

```diff
@@ -12,7 +12,7 @@ const Box = ({ x, y, width, selected, strokeColor, color }) => (
         x={x - 1}
         y={y - 1}
         width={width + 2}
-        stroke="#31BFAE"
+        stroke="#009688"
```

Comment:
You use this color many times. Should it be included in the theme instead? Is there a way to do it?